src/templates/_macros/app_tile.html

```diff
@@ -3,72 +3,79 @@
 {% from "_macros/previews.html" import preview_tray %}
 {% from "_macros/stars.html" import stars %}
 
-{% macro app_tile(app, feed_app=None, is_detail=False, tray=False, src=None) %}
+{% macro app_tile(product, feed_app=None, is_detail=False, tray=False, src=None) %}
```

Comment:
I thought we were going to split this up into one for each content type? I really do not like the jam-everything-into-one-function approach.